core/storage/question/gae_models_test.py

```diff
@@ -183,6 +229,16 @@ def test_create_question_skill_link(self) -> None:
         self.assertEqual(
             questionskilllink_model.skill_difficulty, skill_difficulty)
 
+    def test_get_all_question_ids_linked_to_skill_id(self) -> None:
```

Comment:
This is a pretty weak test. Try and write multiple separate tests that cover the various behavioural cases properly; don't just aim for coverage completion.

Reply:
Sorry I did not cover the case when skill has questions linked to it.
I have added the required test_case and checked both conditions when skill has or has not questions linked to it.